hknweb/settings/common.py

```diff
@@ -11,8 +11,10 @@
 """
 import os
 from django.conf.global_settings import DATETIME_INPUT_FORMATS
+from hknweb.utils import DATETIME_12_HOUR_FORMAT
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+
```

Comment:
Unnecessary line add (not crucial to change, but would be nice)

Reply:
Huh not sure where that came from